src/Sentry/Protocol/ISpan.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+
+namespace Sentry.Protocol
+{
+    public interface ISpan
+    {
+        SentryId SpanId { get; set; }
+
+        SentryId? ParentSpanId { get; set; }
+
+        SentryId TraceId { get; set; }
+
+        DateTimeOffset StartTimestamp { get; set; }
+
+        DateTimeOffset EndTimestamp { get; set; }
```

Comment:
EndTimestamp is Timestamp unless it was changed recently.

Reply:
`timestamp` is the serialized name, I'd name the property `EndTimestamp` not to suck so much from the API point of view

Reply:
What do you mean recently?

Reply:
nothing changed. `EndTimestamp` is the C# type, but serialize this as `timestamp` (which is the sentry event protocol for timestamp)

Reply:
oh right, I was too sleepy to understand code.

Reply:
Ah, I misunderstood the original comment as well.